.taskcluster.yml

```diff
@@ -101,7 +101,7 @@ tasks:
           exit /b 0
         - git rev-parse HEAD > revision.txt
         - set /p revision=< revision.txt
-        - go get -ldflags "-X main.revision=%revision%" -v -u -t ./...
+        - go install -ldflags "-X main.revision=%revision%"
```

Comment:
We still need `./...` to ensure the command line tools are built, and I'd like to keep the `-v` for getting more feedback about the build process.